subprojects/snapshots/src/integTest/groovy/org/gradle/internal/vfs/VirtualFileSystemPartialInvalidationIntegrationTest.groovy

```diff
@@ -24,6 +24,12 @@ class VirtualFileSystemPartialInvalidationIntegrationTest extends AbstractIntegr
 
     private static final INCUBATING_MESSAGE = "Partial virtual file system invalidation is an incubating feature"
 
+    def setup() {
+        executer.beforeExecute {
+            withPartialVfsInvalidation(false)
+        }
+    }
```

Comment:
Can we add a comment here why this is here? Maybe just do the enabling/disabling explicitly at the start of each test instead?

Reply:
I added a comment.